source/assets/components/recipeViewer.js

```diff
@@ -131,7 +159,11 @@ class recipeViewer extends HTMLElement {
 
     this.shadowRoot.querySelector('section').innerHTML = `
     <main id = "card-information">
-        <div id = "recipe-title"></div> 
+        <div id = "recipe-title"></div>
+        <div class = "favorite-button">
+          <img id = "fav-btn" src="./assets/icons/favorite/favorite-blank.png" width = "30px" height = "30px" alt="favorite button">
+          <button>Favorite the Recipe</button>
```

Comment:
Differences in naming conventions may lead to an issue in the future when we merge `favoriteList` branch